packages/terra-brand-footer/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```

Comment:
We need a MVB guide for this

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1167/commits/06ebb6448986df1d498873f7164e9147d4b653c7.